addons/l10n_es_edi_facturae/models/res_partner.py

```diff
@@ -0,0 +1,37 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class Partner(models.Model):
+    _inherit = 'res.partner'
+
+    l10n_es_edi_facturae_person_type = fields.Char(string='Facturae EDI Person Type Code',
+            compute='_compute_l10n_es_edi_facturae_person_type', store=False, readonly=True,)
+    l10n_es_edi_facturae_residence_type = fields.Char(string='Facturae EDI Residency Type Code',
+            compute='_compute_l10n_es_edi_facturae_residence_type', store=False, readonly=True,)
+    l10n_es_edi_facturae_cif_nif_nie = fields.Char(string='Facturae EDI Non-VAT Tax Identifier (CIF/NIF/NIE)')
```

Comment:
Need to see, but normally the vat field itself could contain those numbers as well.  But what is it purpose?